pkg/log/log_file_test.go

```diff
@@ -46,9 +45,13 @@ func initFlags() *pflag.FlagSet {
 
 func restoreFlagDefaultValues() {
 	testFlags.Set(logToStdErrFlag, "true")
+	flag.Set(logToStdErrFlag, "true")
```

Comment:
I am curious about why we have to worry about the global flags now. Does it have anything to do with the klog v1 -> v2 transition? Maybe it would be good to leave a comment.

Reply:
Global flag updates removed.